docs/walk-through/loops.md

```diff
@@ -177,6 +178,32 @@ spec:
       args: ["echo sleeping for {{inputs.parameters.seconds}} seconds; sleep {{inputs.parameters.seconds}}; echo done"]
 ```
 
+## `withSequence` example
+
+This example run a simple template multiple times using `withSequence`.
+
+```yaml
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  generateName: loop-sequence-
+spec:
+  entrypoint: loop-sequence-example
+
+  templates:
+  - name: loop-sequence-example
+    steps:
+    - - name: run-pod
+        template: run-pod
```

Comment:
this should have more specific naming than `run-pod`. if the command is changed to print "hello world", then this should be "hello-world" or similar
```suggestion
    - - name: hello-world-x5
        template: hello-world
```

Reply:
Follow-up comment: https://github.com/argoproj/argo-workflows/pull/12577#discussion_r1498165392